### PR TITLE
Order cards by name in deck-load screen

### LINKED ISSUE #247: Alphabetically sort cards on deck-load screen (cosmetic)
Currently, cards are shown in the reverse order that they were added to the deck.

Example:
token_preview
Those were added to the deck in alphabetical order.

Alphabetically sorting this display won't do much but sorted output is just nicer to look at. It would, at minimum, be quicker to check for cards in *"Is this the right deck?"* situations.

### PR DESCRIPTION
**Pull request by ctrlaltca**
Fix #247

### CODE REVIEW

File: cockatrice/src/deckview.cpp [hunk 1]
```diff
@@ -257,6 +264,7 @@ void DeckViewCardContainer::rearrangeItems(const QList<QPair<int, int> > &rowsAn
         
         QList<QString> cardTypeList = cardsByType.uniqueKeys();
         QList<DeckViewCard *> row = cardsByType.values(cardTypeList[i]);
+        qSort( row.begin(), row.end(), sortCardsByName);
```
Comment (ZeldaZach) [inline cockatrice/src/deckview.cpp]: Minor thing, but there shouldn't be a space between `(` and `row.begin()`.

**Comment by ZeldaZach** [comment]
Works fully as expected and good to go! :shipit:

**Comment by woogerboy21** [comment]
This seems to work for me as well.

**Comment by Fizztastic** [comment]
Tested and works as expected.

**Pull request merged**